docker/databricks/00-custom-spark-driver-defaults-alluxio.conf

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+[driver] {
+ "spark.databricks.delta.preview.enabled" = "true"
+ "spark.locality.wait" = "0s"
```

Comment:
based on our recent testing we should remove locality wait

Reply:
in latest update